contrib/mesos/docs/ha.md

```diff
@@ -0,0 +1,66 @@
+## High Availability
+
+### Scheduler
+
+Implementation of scheduler HA features has started and includes:
```

Comment:
"has started" is confusing -- since the next section says you already allow multiple scheduler instances in failover senario